lib/common-utils.coffee

```diff
@@ -0,0 +1,25 @@
+debug = require('debug')('sphere-product-import-common-utils')
+_ = require 'underscore'
+_.mixin require 'underscore-mixins'
+
+class CommonUtils
```

Comment:
You don't need a class if you just have pure functions. Just export each function separately.